beast/physicsmodel/prior_weights_dust.py

```diff
@@ -85,6 +85,23 @@ def _two_lognorm(xs,
     normalization = np.trapz(pointwise, x=xs)
     return pointwise / normalization
 
+def _exponential(x, a=2.0, N=1.):
+    """
+    Exponential distribution
+    Parameters
+    ----------
+    x: vector
+       x values
+    a: float
+       Single parameter in exp: a*e^-ax
+    N: floats
```

Comment:
Is this parameter needed?  Shouldn't this be set by the desire to have this prior integrate to 1 over the min/max age range?  If so, then it is not needed and should be set inside the function.

Reply:
I was following the format of the other prior choices.  Will leave as is.